src/javascript-node-translation-info-factory.js

```diff
@@ -0,0 +1,35 @@
+const Pofile                = require('pofile');
+const { MARKER_NO_CONTEXT } = require('./constants.js');
```

Comment:
I'm not sure we really want to switch to those non-standards indents, do we ? :) unless you change them withibn the whole codebase ? not sure either !

Reply:
Sorry, this is a matter of habits, I will conform to your standards